src/lib/db/client-applications-store.ts

```diff
@@ -420,4 +421,30 @@ export default class ClientApplicationsStore
             },
         };
     }
+
+    private remapUsageRow = (input) => {
```

Comment:
Added case, when input is list of **projects** instead of just single **project**.